casino/tragaMonedasPlus.js

```diff
@@ -0,0 +1,87 @@
+        console.log("Carrete 2: ", this.figuras[carrete2]);
+        console.log("Carrete 3: ", this.figuras[carrete3]);
+        console.log("Carrete 4: ", this.figuras[carrete4]);
+        console.log("Carrete 5: ", this.figuras[carrete5]);
+        if ((carrete1 == carrete2) && (carrete2 == carrete3) && (carrete3 == carrete4) && (carrete4 == carrete5)) {
+            console.log("Felicitaciones, usted quintuplico el valor de su apuesta, su dinero disponible es: " + this.getpremio());
+            var result = 'Figuras que salieron en la maquina :' + ' ' + (this.figuras[carrete1]) + ' ' + (this.figuras[carrete2]) + ' ' + (this.figuras[carrete3]) + (this.figuras[carrete4]) + ' ' + (this.figuras[carrete5]) + ' ' + 'Todas las figuras son iguales. Gano Jugador' + '\n' + '---------------------' + '\n';
+            fs.appendFile('resultadoTragamonedasPlus.txt', result, function (err) {
+                if (err)
+                    throw err;
+            });
+        }
+        else if ((carrete1 == carrete2) && (carrete2 == carrete3)) {
+            console.log("¡¡Bien hecho!!Usted recupero el dinero de su apuesta", this.apuesta);
+            var result = 'Figuras que salieron en la maquina :' + ' ' + (this.figuras[carrete1]) + ' ' + (this.figuras[carrete2]) + ' ' + (this.figuras[carrete4]) + '' + (this.figuras[carrete5]) + ' ' + 'Salieron tres figuras iguales consecutivas. El jugador recupera su apuesta' + '\n' + '---------------------' + '\n';
+            fs.appendFile('resultadoTragamonedasPlus.txt', result, function (err) {
+                if (err)
+                    throw err;
+            });
+        }
+        else if ((carrete2 == carrete3) && (carrete3 == carrete4)) {
+            console.log("¡¡Bien hecho!! Usted recupero el dinero de su apuesta", this.apuesta);
+            var result = 'Figuras que salieron en la maquina :' + ' ' + (this.figuras[carrete1]) + ' ' + (this.figuras[carrete2]) + ' ' + (this.figuras[carrete4]) + '' + (this.figuras[carrete5]) + ' ' + 'Salieron tres figuras iguales consecutivas. El jugador recupera su apuesta' + '\n' + '---------------------' + '\n';
+            fs.appendFile('resultadoTragamonedasPlus.txt', result, function (err) {
+                if (err)
+                    throw err;
+            });
+        }
+        else if ((carrete3 == carrete4 && carrete4 == carrete5)) {
+            console.log("¡¡Bien hecho!! Usted recupero el dinero de su apuesta", this.apuesta);
+            var result = 'Figuras que salieron en la maquina :' + ' ' + (this.figuras[carrete1]) + ' ' + (this.figuras[carrete2]) + ' ' + (this.figuras[carrete4]) + '' + (this.figuras[carrete5]) + ' ' + 'Salierom tres figuras iguales consecutivas. El jugador recupera su apuesta' + '\n' + '---------------------' + '\n';
+            fs.appendFile('resultadoTragamonedasPlus.txt', result, function (err) {
+                if (err)
+                    throw err;
+            });
+        }
```

Comment:
Al string result le falta informar sobre this.figuras[carrete3] en cada uno de los 3 elseif statements.

Otra cosa a mejorar, ya que los 3 hacen lo mismo, podría ser hacer un solo bloque elseif poniendo las 3 condiciones dentro de la condicion, separando con ||